.github/workflows/ci.yaml

```diff
@@ -0,0 +1,99 @@
+name: CI
+
+on:
+  merge_group:
+  pull_request:
+      - 'DCO'
+      - 'LICENSE'
+      - 'README.md'
+    branches:
+      - "main"
+      - "v**"
+  workflow_dispatch:
+
+jobs:
+  static-security-analysis:
+    runs-on: ubuntu-22.04
+    env:
+      GO111MODULE: on
+    steps:
+      - name: Checkout Source
+        uses: actions/checkout@v3
+      - name: Run Gosec Security Scanner
+        uses: securego/gosec@master
+        with:
+          args: -exclude-dir e2etest -severity medium ./...
+
+      - name: Golang Vulncheck
+        uses: Templum/govulncheck-action@v0.10.1
+        with:
+          skip-upload: true
+
+  commitlint:
+    runs-on: ubuntu-22.04
+    steps:
+      - uses: actions/checkout@v3
+        with:
+          fetch-depth: 0
+      - uses: wagoid/commitlint-github-action@v5
+
+  unit-tests:
+    runs-on: ubuntu-22.04
+    steps:
+      - name: Cancel Previous Runs
+        uses: styfle/cancel-workflow-action@0.9.1
+        with:
+          access_token: ${{ github.token }}
+
+      - name: Checkout
+        uses: actions/checkout@v3
+
+      - uses: actions/setup-go@v4
+        with:
+          go-version-file: "go.mod"
+          cache: true
+
+      - name: Install Dependencies
+        run: |
+          sudo apt-get update
+          sudo apt-get install python3-pip arping ndisc6
+          sudo pip3 install invoke semver pyyaml
+
+      - name: Unit Tests
+        run: |
+          inv test
+
+      - name: Lint
+        run: |
+          ENV=host make lint
+          make checklicense
+          go mod tidy
+          make checkuncommitted
```

Comment:
if I understood your comment (and if we drop this line) it might be less obvious to understand what to fix
On the other hand, if we change checklicense to bumplicense, we might want to add another one so we intercept the license failure.

Reply:
my rational is that if multiple steps fail we should not have to do multiple ci cycles to figure it out, is there an option we have a single command that "bumps"(fixes) everything? I don't think it's too important to have them separately so it's obvious what fails, when the solution is running some make and committing blindly :sweat_smile:

Reply:
done